### DISCUSSION TITLE
android app can not login anymore

### DISCUSSION THREAD

**Discussion by heartacker** [community/community#65017] [Mobile] [answered]
### Select Topic Area

Bug

### Body

You are being redirected to the authorized application.
If your browser does not redirect you back, please click here to continue.

![img_v2_8a3403ec-3bf4-49b0-9a74-bf1376e2a48g](https://github.com/community/community/assets/1876302/8d4e731a-0b78-476c-8fc6-5f1d029196da)

![img_v2_9048036e-b35d-4103-a865-f1ad83dcf3ag](https://github.com/community/community/assets/1876302/0d1c1751-a759-434f-8b1e-b72a9f5aa446)

**Comment by fraimparato** [comment]
I have the same issue: stuck on the same message. It seems like the link to the app does not work.

**Comment by fraimparato** [comment, marked as answer]
I was searching for a solution and I found this thread, it worked for me: https://github.com/orgs/community/discussions/64844

**Comment by alcere** [comment]
We are sorry for the troubles here :(. As @fraimparato shared, Brave browser blocked all app redirects by default in its latest update and that's blocking our the sing-in flow, please give it a try again following the steps described [here](https://github.com/orgs/community/discussions/64844). Thanks!
